fe/fe-core/src/main/java/com/starrocks/connector/Connector.java

```diff
@@ -2,7 +2,21 @@
 
 package com.starrocks.connector;
 
+import java.util.Map;
+
 public interface Connector {
+    class Context {
```

Comment:
why use inner class?

Reply:
just some personal habit, do you think we should move it out?

Reply:
there is a little over design.

Reply:
where this context will be used?

Reply:
ConnectorFactory

Reply:
There is no sence using properties. Connector will init different field with different type.

Reply:
what will be passed to init Connector

Reply:
@caneGuy is it right?

Reply:
> 
eg:
`HiveConnector` need a field like `hmsUris` .

Reply:
> @caneGuy is it right?

+1

Reply:
this is passing by properties